### PR TITLE
Feature/12526 separate debugger install instructions

### PR DESCRIPTION
**Pull request by Cortex-DMS**

### CODE REVIEW

File: content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md [hunk 1]
```diff
@@ -181,7 +31,7 @@ To install the components required for debugging, perform the steps detailed in
     -ServiceFabricUsingSelfSignedCertificates $false `
     -ServiceFabricApiGatewayBasicAuthUsername "BasicAuthUser" `
     -ServiceFabricApiGatewayBasicAuthPassword '#_065077199197085!212123173135087074174142102155007175102029143220132038175026114248243207204119030125106032237087162060168108135168241247037070081~187087056217118!069132229129134129097089241180163#' `
-    -DotNetFlowDebuggerEndpoint "https://server.domain.com:8722/api/" `
+    -DotNetFlowDebuggerEndpoint 'https://server.domain.com:8722/api/' `
```
Comment (cortex-lp) [inline content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md:34]: do we need to do that for all params?

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/assign-certificate-friendly-name.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+1. Click the Windows button (`Start`).
+1. Type `certlm.msc` and press `Enter` to open the Certificate Manager dialog.
+1. Expand `Personal` and select `Certificates`.
+1. You should see your certificate in this store.
+1. Right-click on the certificate and select `Properties`.
+1. On the `General` tab in the `Friendly Name` text box, enter a name to be used for the certificate.
+1. Click `OK`.
+
+[Configure CORTEX Gateway Installation Script]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.ConfigureCortexGatewayInstallationScriptNew" >}}
```
Comment (cortex-lp) [inline content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/assign-certificate-friendly-name.md:11]: multi only?

Comment (cortex-lp) [reply content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/assign-certificate-friendly-name.md:11]: for all multi only, make it in multi folder

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/licensing.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Ensure that a valid {{% ctx %}} licence file named `Cortex.lic` exists on the Web Application server, in the location `%ProgramData%\Cortex\Licences`. If it does not, follow the instructions located at [Licensing Requirements][].
+
+[Licensing Requirements]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.LicensingRequirements" >}}
```
Comment (cortex-lp) [inline content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/licensing.md:3]: multi only?

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/configure-debugger-installation-script.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    {{% alert title="Note" %}}
+More advanced configuration (such as changing ports) can be undertaken by modifying the `Cortex.Innovation.Install.Config.json` file but this shouldn't be required for most installations. More information about this can be found at {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.Advanced.AdvancedConfigSingleServer" title="Advanced Application Server and Load Balancer Configuration Changes" >}}.
+    {{% /alert %}}
+
+1. Save and close `Cortex.Innovation.Install.ps1`.
+
+[Certificate Requirements]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.CertificateRequirements" >}}
+[CORTEX Encrypted]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.Advanced.EncryptText" >}}
+[Encryption Requirements]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.EncryptionRequirements" >}}[Install Gateway]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.ConfigureCortexGatewayInstallationScriptNew" >}}
```
Comment (cortex-lp) [inline content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/configure-debugger-installation-script.md:77]: multi only?

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/assign-certificate-friendly-name.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+1. Click the Windows button (`Start`).
+1. Type `certlm.msc` and press `Enter` to open the Certificate Manager dialog.
+1. Expand `Personal` and select `Certificates`.
+1. You should see your certificate in this store.
+1. Right-click on the certificate and select `Properties`.
+1. On the `General` tab in the `Friendly Name` text box, enter a name to be used for the certificate.
+1. Click `OK`.
+
+[Configure CORTEX Gateway Installation Script]: {{< url path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.ConfigureCortexGatewayInstallationScriptNew" >}}
```
Comment (cortex-lp) [inline content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/assign-certificate-friendly-name.md:11]: multi only?

File: content/en/docs/2023.11/getting-started/on-premise/add-innovation-to-72/multiple-server-with-ha/install-application-and-loadbalancer-servers.md [hunk 6]
```diff
@@ -133,20 +133,20 @@ To check all necessary ports are free, follow these steps:
...
     -BlockPackagesPath "C:\Install\Cortex Innovation {{< version >}} - Block Packages.zip" `
     -ApiGatewayBasicAuthUsername "BasicAuthUser" `
-    -ApiGatewayBasicAuthPassword "#_065077199197085!212123173135087074174142102155007175102029143220132038175026114248243207204119030125106032237087162060168108135168241247037070081~187087056217118!069132229129134129097089241180163#" `
+    -ApiGatewayBasicAuthPassword '#_065077199197085!212123173135087074174142102155007175102029143220132038175026114248243207204119030125106032237087162060168108135168241247037070081~187087056217118!069132229129134129097089241180163#' `
     -CustomerName "Customer1" `
     -ApplicationServerIPv4Addresses @("192.168.1.1", "192.168.1.2", "192.168.1.3") `
     -LoadBalancerServerIPv4Address "192.168.1.4" `
     -ServerCertificatePath "C:\Install\Certificates\cert.pfx" `
-    -ServerCertificatePassword "#_186224203199039!168155035100255155162114088105027~037077176064169!055142133197216213014226219148063#" `
+    -ServerCertificatePassword '#_186224203199039!168155035100255155162114088105027~037077176064169!055142133197216213014226219148063#' `
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/add-innovation-to-72/multiple-server-with-ha/install-application-and-loadbalancer-servers.md:141]: To align with Code file

File: package-lock.json [hunk 7]
```diff
@@ -55,9 +55,9 @@
       "dev": true
     },
     "node_modules/ansi-regex": {
-      "version": "5.0.0",
-      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-5.0.0.tgz",
-      "integrity": "sha512-bY6fj56OUQ0hU1KjFNDQuJFezqKdrAyFdIevADiqrWHwSlbmBNMHp5ak2f40Pm8JTFyM2mqxkG6ngkHO11f/lg==",
+      "version": "5.0.1",
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply package-lock.json:58]: agreed to leave

File: content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/setup-gateway.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ weight: 50
 
 # {{% param title %}}
 
-This guide describes how to setup Gateway. Please ensure that [Install Web Application Server][] has been completed before taking these steps.
+This guide describes how to setup Gateway. Please ensure that [Perform Gateway Installation][] has been completed before taking these steps.
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/setup-gateway.md:10]: reworded

File: content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md (hunk 1 above)
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md:34]: no only passwords should be single quotes - have replaced this one with double

File: content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md [hunk 9]
```diff
@@ -264,76 +114,26 @@ To install the components required for debugging, perform the steps detailed in
...
+{{< section "/install-web-application-server/grant-gateway-user-additional-folder-permissions.md">}}
 
 ## Preserve installation files
 
-Ensure that the installation files are backed up or kept on the server, especially the scripts and config files that have been modified. This will make it easier to perform further actions in future, such as troubleshooting, certificate rollover, uninstallation, reinstallation and updates.
+{{< section "/preserve-installation-files.md">}}
 
 ## Next Steps?
 
 1. [Setup {{% ctx %}} Gateway][Setup CORTEX Gateway]
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md:125]: Changed

File: content/en/docs/2023.11/getting-started/on-premise/add-innovation-to-72/multiple-server-with-ha/install-web-application-server/perform-debugger-installation.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+{{< section "/install-application-server/single-server/check-application-services.md">}}
+
+# Preserve installation files
+
+{{< section "/preserve-installation-files.md">}}
+
+## Next Steps?
+
+1. [Perform Gateway Installation][]
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/add-innovation-to-72/multiple-server-with-ha/install-web-application-server/perform-debugger-installation.md:56]: fixed

File: content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md [hunk 11]
```diff
@@ -198,7 +48,7 @@ To install the components required for debugging, perform the steps detailed in
     *>&1 | Tee-Object -FilePath "cortex-gateway-install-log.txt"
     ```
 
-    {{% alert title="Important" color="warning" %}}Parameters required to be {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.Advanced.EncryptText" title="CORTEX Encrypted" >}} must be encrypted on one of the servers specified in the {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.MultipleServerWithHA.EncryptionRequirements" title="Encryption Requirements" >}} steps.{{% /alert %}}
+    {{% alert title="Important" color="warning" %}}Parameters required to be {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.Advanced.EncryptText" title="CORTEX Encrypted" >}} must be encrypted on one of the servers specified in the {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.SingleServerWithoutHA.EncryptionRequirements" title="Encryption Requirements" >}} steps.{{% /alert %}}
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/getting-started/on-premise/install-innovation-only/multiple-server-with-ha/install-web-application-server/perform-gateway-installation.md]: resolved

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/licensing.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+Ensure that a valid {{% ctx %}} licence file named `Cortex.lic` exists on the Web Application server, in the location `%ProgramData%\Cortex\Licences`. If it does not, follow the instructions located at [Licensing Requirements][].
+
+[Licensing Requirements]: {{< url path="Cortex.GettingStarted.OnPremise.AddInnovationTo72.MultipleServerWithHA.LicensingRequirements" >}}
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/licensing.md:3]: single doesn't used shared section

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/configure-debugger-installation-script.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+    The `ApiGatewayBasicAuthUsername` and `ApiGatewayBasicAuthPassword` will be needed later, [when upgrading Gateway][Install Gateway].
+
+    {{% alert title="Note" %}}
+More advanced configuration (such as changing ports) can be undertaken by modifying the `Cortex.Innovation.Install.Config.json` file but this shouldn't be required for most installations. More information about this can be found at {{< ahref path="Cortex.GettingStarted.OnPremise.InstallInnovationOnly.Advanced.AdvancedConfigSingleServer" title="Advanced Application Server and Load Balancer Configuration Changes" >}}.
+    {{% /alert %}}
+
+1. Save and close `Cortex.Innovation.Install.ps1`.
+
+[Certificate Requirements]: {{< url path="Cortex.GettingStarted.OnPremise.AddInnovationTo72.MultipleServerWithHA.CertificateRequirements" >}}
```
[the remark this answers is not in the document]
Comment (Cortex-DMS) [reply content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-to-7.2/configure-debugger-installation-script.md:75]: single doesn't used shared section

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/licensing.md (hunk 3 above)
Comment (Cortex-DMS) [reply content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/licensing.md:3]: single doesn't used shared section

File: content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/configure-debugger-installation-script.md (hunk 4 above)
Comment (Cortex-DMS) [reply content/en/docs/2023.11/_shared/install-web-application-server/add-innovation-only/configure-debugger-installation-script.md:77]: single doesn't used shared section

**Review by cortex-lp — APPROVED** [review]

**Pull request merged**